service/src/main/kotlin/io/provenance/explorer/domain/models/explorer/ExplorerModels.kt

```diff
@@ -28,3 +30,126 @@ data class GasStatistics(
     val maxGasPrice: Int,
     val averageGasPrice: BigDecimal
 )
+
+data class Params(
+    val cosmos: CosmosParams,
+    val prov: ProvParams,
+)
+
+data class CosmosParams(
+    val authParams: AuthParams,
```

Comment:
Update the field names match [query-commands](https://docs.provenance.io/blockchain/using-provenance/query-command#query-command)

- auth
- bank
- distribution
- gov
- mint
- slashing
- staking
- ibc